src/main/java/com/knu/buga1chuk/serialization/interfaces/PersonSerializationService.java

```diff
@@ -0,0 +1,16 @@
+package com.knu.buga1chuk.serialization.interfaces;
+
+import com.knu.buga1chuk.model.Person;
+
+import java.io.File;
+import java.util.List;
+
+public interface PersonSerializationService {
```

Comment:
Move this interface to the service folder

Reply:
fix